pywps/ext_autodoc.py

```diff
@@ -8,6 +8,21 @@
 from pywps.app.Common import Metadata
 
 
+class MetadataUrl(Metadata):
```

Comment:
I would add the `anonymous` option to the `Metadata` class itself instead of sub-classing it. What do you think?

Reply:
That was my initial thought until I saw how the `Metadata` class is being used.  It's being used to generate the xml response, it can be serialized to and deserialized from json format.  So bigger impact and not required for what I needed to solve.

So I am unsure if adding the new `anonymous` option there is appropriate.

But it's my first pywps commit so I will follow you guidance.  Let me know if adding it to the `Metadata` is the right thing to do.

Reply:
@tomkralidis What is your opinion?